cypress/e2e/data-stream.cy.js

```diff
@@ -0,0 +1,59 @@
+import generateUniqueName from '../support/utils'
+import selectors from '../support/selectors'
+
+let dataStreamName
+
+describe('Data Stream spec', () => {
+  beforeEach(() => {
+    dataStreamName = generateUniqueName('DataStream')
+
+    cy.login()
+    cy.openProductThroughSidebar('data-stream')
+  })
+
+  it('should create and delete a data stream with the standard connector', () => {
+    // Arrange
+    cy.get(selectors.dataStream.createButton).click()
+
+    // Act
+    cy.get(selectors.dataStream.nameInput).type(dataStreamName)
+
+    cy.get(selectors.dataStream.sourceDropdown).should('have.text', 'Edge Applications')
+    cy.get(selectors.dataStream.templateDropdown).should('have.text', 'Template teste')
+    cy.get(selectors.dataStream.editorBody).should('not.be.empty')
```

Comment:
Eu não usaria asserções na parte de ação, pois polui o teste.

Se quiser fazer um it específico apenas para verificar os defaults, poderia inclusive ser um e2mock, pois não envolve comunicação com a API.

Reply:
Vale para as outras asserções na sessão Act.